src/test/run-make/issue-30063/Makefile

```diff
@@ -0,0 +1,21 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) -C codegen-units=4 -o foo-output foo.rs
+	ls foo-output
```

Comment:
I may not remember the current directory of run-make tests, but I think this puts files in the source tree? Adding `$(TMPDIR)` on front should fix it.

I also tend to use `rm` to ensure that the file was created so that way it also doesn't interfere with further tests, but this should work either way.